CHANGELOG.md

```diff
@@ -6,4 +6,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-[Unreleased]: https://github.com/MetaMask/auto-changelog/
+## [1.0.0]
+### Added
+- The initial `auto-changelog` implementation, adapted from the original `auto-changelog.js` script in  `metamask-extension` ([#8](https://github.com/MetaMask/auto-changelog/pull/8)).
```

Comment:
It seemed silly to have "changed" or "fixed" categories, since _nothing_ has been changed since the previous release (which does not exist). So instead I summarized everything as one point, linking the most relevant PRs.